app/src/main/java/com/github/se/cyrcle/ui/map/overlay/ZoomControls.kt

```diff
@@ -26,17 +25,16 @@ fun ZoomControls(onZoomIn: () -> Unit, onZoomOut: () -> Unit) {
         modifier =
             Modifier.padding(15.dp)
                 .background(Color.Black.copy(alpha = 0.70f), shape = RoundedCornerShape(8.dp))
-                .sizeIn(maxWidth = 35.dp)) {
-          Box(modifier = Modifier.testTag("ZoomControlsIn")) {
```

Comment:
I would have appreciated if you mentioned the UI changes in your PR description too. And even better if you did a small PR just for that, (Remember one task per PR)

Reply:
Sorry I didn't really consider it as UI since the change didn't affect it... :sweat_smile:

Reply:
I see the reasoning